tdrs-backend/docs/api/set_profile.md

```diff
@@ -1,12 +1,12 @@
 
```

Comment:
Updating documentation to reflect the change from a `POST` endpoint to a `PATCH` endpoint.